code/game/machinery/vending_types.dm

```diff
@@ -719,6 +719,8 @@
 	density = FALSE //It is wall-mounted, and thus, not dense. --Superxpdude
 	vend_id = "meds"
 	products = list(
+		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 3,
+		/obj/item/reagent_containers/hypospray/autoinjector/coagzolug = 3,
```

Comment:
I think this should be a new subtype, and put in locations where it's actually needed

or even more ideally, the new subtype could be `/expedition` and put on horizon shuttles, and have coagzolug and peridaxon (for popped lungs)

cause these nanomeds are used like everywhere, all over horizon, on offships, random away sites, etc, I don't think all of them should have coagzolug

Reply:
actually there's also `/obj/machinery/vending/wallmed2` and they're both very inconsistently used and really need a refactor

not requesting a refactor though, but IMO coagzolug shouldn't be added to all nanomeds

Reply:
It doesn't seem too bad to me? If I'm reading the code correctly, coagzolug seems to reduce external bleeding by 20%. The only situation I see it being remarkably useful is when you're a long way from atmosphere and you're bleeding inside your voidsuit, which I don't see any issue with making accommodations for.

I also don't think we need another subtype of the NanoMed. We should be trying to aim for as few subtypes as possible imo, the fewer we have the easier it is to maintain them and the less annoying it is to memorise them while mapping.